requirements.txt

```diff
@@ -1,5 +1,5 @@
 talisker[gunicorn]==0.15.0
-canonicalwebteam.discourse-docs==0.6.5
+canonicalwebteam.discourse-docs==0.10.3
```

Comment:
This version does not exist.

Reply:
As mentioned in the description. Waiting for the PR to be merged and then we can qa the docs.

Adding the don't merge label

Reply:
The PR was merged and the package deployed.